UnityProject/Assets/Scripts/Objects/VisibleBehaviour.cs

```diff
@@ -96,4 +106,9 @@ private void UpdateState(bool _aliveState)
 			}
 		}
 	}
+
+	private bool canBeDisabled(MonoBehaviour[] scripts, int i)
+	{
+		return !neverDisabled.Contains(scripts[i].GetType().Name);
+	}

```

Comment:
wouldn't it be easier to just call `!neverDisabled.Contains(scripts[i].GetType().Name);` in the `UpdateState` method?

or at least just passing the current script instead of the array and the index.

Reply:
it was quick-extracted for the sake of better readability, but you're right about passing the script